### PR TITLE
Add Windows instructions

### PR DESCRIPTION
**Pull request by Alchus**
Hey! The pucatrade subreddit may not love bots, but hey, not everybody can be cool. I was able to install and run the script on Windows, and added instructions for that to the Readme. 

Also added a tiny tweak: the program should fail to assert a successful login (and thus halt) if we're not redirected to the dashboard after logging in.

I look forward to making this software as useful as possible! My priority will be adding logic to handle searching for "bundle" trades of lower value cards while ignoring matches for single, inexpensive cards.

By the way: it's my first time working in Python, but how hard can it be?

### CODE REVIEW

**Comment by tomreece** [comment]
Awesome thanks for the first pull request. I'm adding some comments now that you'll need to fix before I can merge it.

File: pucauto.py [hunk 1]
```diff
@@ -46,6 +46,7 @@ def confirm_trade():
 
 def main():
     login()
+    assert 'pucatrade.com/dashboard' in browser.title
```
Comment (tomreece) [inline pucauto.py:49]: There is no `browser.title`. You'd want `driver.title` instead. However, it looks like you're asserting on the url. The title will be "Dashboard |  PucaTrade - Trade Magic: The Gathering Cards Online".

So you can use `driver.current_url` for this assertion.

**Comment by tomreece** [comment]
In the README, under the Installation header can you turn it into 2 main sections: Mac and Windows 

This makes it easier to read.

Like this:

```
### Installation

#### Mac

##### Prerequisites

1. Git
2. Python
3. Pip
4. Firefox

##### Steps

1. Clone this repository with `git clone https://github.com/tomreece/pucauto.git` or download and unzip the .zip using the link over on the right of this page.
2. Open Terminal.
3. Go to the pucauto folder you just cloned or downloaded.
3. Run `sudo pip install -r requirements.txt` and enter your system password when prompted.

#### Windows

##### Prerequisites

1. Python 3
2. Selenium
3. Firefox

##### Steps

1. Download this repository using the link on the right -->
2. Use 7zip or WinRar to expand the archive
3. Download Python from https://www.python.org/downloads/.
4. Install Python. IMPORTANT: Select the check box to associate Python with environment variables
5. Download and install Firefox if you don't already have it. https://www.mozilla.org/en-US/
6. Download Selenium from https://pypi.python.org/pypi/selenium. You want the file named selenium-2.48.0.tar.gz
7. Extract the Selenium archive. Extract the nested archive /dist/selenium-2.48.0/.
8. From inside the interior folder (you should see sub-folders namd build, dist, py, and selenium.egg), Shift-Right Click, then select 'Open command window here'.
9. Run the command "python setup.py install"
```

**Comment by tomreece** [comment]
Are you going to reopen a new pull request?

**Comment by Alchus** [comment]
Yeah, for just the Windows instructions. 
Usually I let NetBeans do the heavy lifting for version control, but as it's my first dabble in Python, I've just been winging it with the github client and notepad++ so far- not as easy to shelve changes to go back to older revisions. I figured it was easier to make a new pull later.

**Comment by tomreece** [comment]
Sounds good. :) Thanks.

**Comment by tomreece** [comment]
@Alchus Sorry to bother you, but people keep asking me for Windows instructions. I'm anxiously awaiting your new pull request that just adds them to the README. I would copy them in from your first pull request, but I want you to get credit as the collaborator. Are you able to submit a new pull request soon? Thanks!

**Comment by Alchus** [comment]
Go ahead and copy them- I'm on the road and its stuck on my home PC

Sorry to have left you hanging

**Comment by tomreece** [comment]
No problem at all!

**Pull request closed without merging**